doc/source/conf.py

```diff
@@ -30,6 +30,7 @@
 extensions = ['sphinx.ext.autodoc',
               'sphinx.ext.mathjax',
               'numpydoc',
+              'sphinx.ext.autodoc',
```

Comment:
`sphinx.ext.autodoc` looks like it is already include as the first item in the list on line 30.

Reply:
thanks Jarrod!